articles/azure-maps/migrate-from-bing-maps-web-services.md

```diff
@@ -468,38 +467,37 @@ For example, in Azure Maps, a blue line with 50% opacity and a thickness of four
 |-------------------------|-------------------------------------------------------------|
-| `origins`               | `origins` – specify in the POST request body as GeoJSON.    |
-| `destinations`          | `destination` – specify in the POST request body as GeoJSON. |
+| `origins`               | `origins` – specify in the `POST` request body as GeoJSON.    |
+| `destinations`          | `destination` – specify in the `POST` request body as GeoJSON.|
 | `endTime`               | `arriveAt`                                                  |
 | `startTime`             | `departAt`                                                  |
 | `travelMode`            | `travelMode`                                                |
 | `resolution`            | N/A                                                         |
 | `distanceUnit`          | N/A – All distances in meters.                              |
 | `timeUnit`              | N/A – All times in seconds.                                 |
-| `key`                   | `subscription-key` – See also the [Authentication with Azure Maps](./azure-maps-authentication.md) documentation. |
-| `culture` (`c`)         | `language` – See [supported languages](./supported-languages.md) documentation.  |
-| `userRegion` (`ur`)     | `view` – See [supported views](./supported-languages.md#azure-maps-supported-views) documentation.     |
+| `key`                   | `subscription-key` – For more information, see [Authentication with Azure Maps]. |
+| `culture` (`c`)         | `language` – For more information, see [Localization support in Azure Maps].  |
+| `userRegion` (`ur`)     | `view` – For more information, see [Azure Maps supported views].     |
 
 > [!TIP]
 > All the advanced routing options available in the Azure Maps routing API (truck routing, engine specifications, avoid…) is support in the Azure Maps distance matrix API.
 
 ## Calculate an isochrone
 
-Azure Maps provides an API for calculating an isochrone, a polygon covering an area that can be traveled to in any direction from an origin point within a specified amount of time or amount of fuel/charge. The Azure Maps route range API is comparable to the isochrone API in Bing Maps;
+Azure Maps provides an API for calculating an isochrone, a polygon covering an area that can be traveled to in any direction from an origin point within a specified amount of time or amount of fuel/charge. The Azure Maps route range API is comparable to the isochrone API in Bing Maps.
 
-* [Route](/rest/api/maps/route/getrouterange) Range**: Calculate a polygon covering an area that can be traveled to in any direction from an origin point within a specified amount of time, distance, or amount of fuel/charge available.
+* [Route] Range**: Calculate a polygon covering an area that can be traveled to in any direction from an origin point within a specified amount of time, distance, or amount of fuel/charge available.
```

Comment:
In line 500, "Range**" has rogue double asterisks. Should this list intro be formatted in bold?